### PR TITLE
Added html to database and rendered the html

### PR DESCRIPTION
**Pull request by ikaur1718**
Updated the database file. Added the html format in the database. Rendered the html from main file.

### CODE REVIEW

**Comment by ikaur1718** [comment]
@benmangold @jsangiolo92 @kennisilverio 
Please review and reflect!

File: database-mongo/feedData.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    boxContent: Array,
+    descriptionHeader: Array,
+    description: Array,
+    imageHeader: Array,
+    images: Array,
+    shippingDate: String,
+    details: Array,
+    html: String
+  });
+
```
Comment (benmangold) [inline database-mongo/feedData.js]: I think this schema and it's model should still be in the model file.  Maybe export them from Model/index.js and import them here instead?

File: database-mongo/index.js [hunk 2]
```diff
@@ -10,59 +9,75 @@ db.on('error', function() {
...
-    })
-  })
-}
+// var seedDatabase = () => {
+//   data.file.forEach((prod) => {
+//     saveDescription(prod, () => {
+//       console.log('Inserted');
+//     })
+//   })
+// }
```
Comment (benmangold) [inline database-mongo/index.js]: lots of commented out code here!  let's get the schema and model back here and export them for the seed file

File: database-mongo/feedData.js (hunk 1 above)
Comment (ikaur1718) [reply database-mongo/feedData.js]: sure i can do that!

File: react-client/src/components/Gallery.jsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+const Gallery = (props) => {
+    return (
+        <img alt="Massdrop x Fostex TR-X00 Purpleheart Headphones" src="https://helpdesk.massdrop.com/hc/en-us/articles/216723438"/>
+    )
+  }
+  
+  export default Gallery;
```
Comment (benmangold) [inline react-client/src/components/Gallery.jsx:8]: this works as a placeholder for the gallery, still thinking through exactly how we will integrate my component into your content.  we can't use this directly in the browser so I think we will have to assemble our own description html get them working together

Comment (ikaur1718) [reply react-client/src/components/Gallery.jsx:8]: I think once we are done working on our own components, we'll talk to joe about that?

File: react-client/src/index.jsx [hunk 4]
```diff
@@ -79,17 +75,60 @@ class App extends React.Component {
...
+        return this.slice(0, idx) + str + this.slice(idx + Math.abs(rem));
+      };
+    
+      htmlCopy = htmlCopy.splice(index, 0, this.state.images[i]);
+      i++;
+    // }
+    this.setState({
+      html: htmlCopy
+    })
+    //console.log(index);
```
Comment (benmangold) [inline react-client/src/index.jsx:110]: I don't think we want to do any manipulation of this HTML once it in the database.  Your component is doing it's job of rendering the stored html.  We have to figure out our process for creating description html which will incorporate my component

Comment (ikaur1718) [reply react-client/src/index.jsx:110]: I was trying to put images dynamically instead of hard coding them

Comment (ikaur1718) [reply react-client/src/index.jsx:110]: Which works, i is just making the it a slow process.

**Review by benmangold — COMMENTED** [review]
Nice work!  Rendering the html stored in the db is a big hurdle.  I don't think we should be doing any processing of it at all once it has been stored.  Rather, we will have to assemble our own description html using the Image Gallery, transpile it to regular html from react jsx, and then store it in the db.

File: react-client/src/index.jsx (hunk 4 above)
Comment (benmangold) [reply react-client/src/index.jsx:110]: gotcha, I'm not sure what the best way to go is

File: react-client/src/components/Gallery.jsx (hunk 3 above)
Comment (benmangold) [reply react-client/src/components/Gallery.jsx:8]: sounds good!

**Comment by benmangold** [comment]
Excellent!  Ready to merge

**Pull request merged**